raiden/network/rpc/client.py

```diff
@@ -310,13 +310,15 @@ def token(self, token_address):
 
         return self.address_token[token_address]
 
-    def discovery(self, discovery_address):
+    def discovery(self, discovery_address, gasprice, startgas):
```

Comment:
It doesn't make sense to add these arguments to only one of the proxies factories, and we should use gas estimation instead of startgas. My suggestion for now is:

- add in constant.py a new value for `registerEndpoint` gas usage, it doesn't have any loops so the required gas is constant.
- create a test that register a few end points and assert on the gas used.
- check the account has balance to cover the constant amount for the default gas price.